src/core/sync/mutex.d

```diff
@@ -374,11 +374,11 @@ unittest
     // by checking that locking is not possible. This assumes
     // that the underlying implementation is well behaved
     // and makes the object non-lockable upon destruction.
-    // For example, Bionic doesn't appear to do so, so this test is
-    // not run on Android.
+    // BTW: Bionic/Musl and DragonFlyBSD don't (appear) to do so, so this test is
+    // not skipped on/for them.
```

Comment:
Change to 'The Bionic and Musl C runtimes and DragonFly don't appear to do so, so skip this test.`